### PR TITLE
INTYGFV-12452: Harmonization of logback file usage and structure

### PR DESCRIPTION
**Pull request by mikeyOne**
Work on this jira was aimed at harmonization of logback usage across all applications and included actions in infra, common, intygsadmin, intygstjanst, logsender, minaintyg, privatlakarportal, rehabstod, statistik and webcert. Briefly, logging functionality was collected in infra/monitoring, a logback-dev file was introduced for use in dev-environment and logback-ocp files (in web/resources as a backup and in devops/openshift/test) was set to produce logging identical to current prod-logging.  A summary of all actions can be found in PRs in common and infra.

### CODE REVIEW

File: devops/openshift/test/config/logback-ocp.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!--
-       ~ Copyright (C) 2019 Inera AB (http://www.inera.se)
+  ~ Copyright (C) 2019 Inera AB (http://www.inera.se)
```
Comment (mww-sopra-steria) [inline devops/openshift/test/config/logback-ocp.xml:3]: Outdated copyright year :-)

File: web/src/main/resources/logback-ocp.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <!--
-       ~ Copyright (C) 2019 Inera AB (http://www.inera.se)
+  ~ Copyright (C) 2019 Inera AB (http://www.inera.se)
```
Comment (mww-sopra-steria) [inline web/src/main/resources/logback-ocp.xml:3]: Outdated Copyright year :-)

File: devops/openshift/test/config/logback-ocp.xml (hunk 1 above)
Comment (mikeyOne) [reply devops/openshift/test/config/logback-ocp.xml]: Done

File: web/src/main/resources/logback-ocp.xml (hunk 2 above)
Comment (mikeyOne) [reply web/src/main/resources/logback-ocp.xml]: Done

**Review by peterssond — APPROVED** [review]

**Pull request merged**